app/javascript/lib/plot.js

```diff
@@ -209,18 +228,7 @@ PlotUtils.getColorForLabel = function(label, customColors={}, editedCustomColors
 
 /** Returns an array of labels, sorted in the order in which they should be plotted (last is 'on top') */
 PlotUtils.getPlotSortedLabels = function(countsByLabel, activeTraceLabel) {
```

Comment:
Any reason why we need to now keep an empty function?

Reply:
good catch, refactoring remnant, will remove